### ISSUE TITLE
I tried to pick-up objects and it didn't work out.

### ISSUE THREAD

**Issue by volcanoqq** [enable3d/enable3d#366]
https://github.com/enable3d/enable3d/assets/113505790/491293fd-a9f9-4f13-b4ff-27d31a351b31

Like in the video, my manipulator tries to grasp the object, but visually they do not collide, why does the object bounce away.

**Comment by volcanoqq** [comment]
`

      // Mechanical arm
      const left = editor.scene.getObjectByName("左抓手");
      const right = editor.scene.getObjectByName("右抓手");

      physicsManager.createRigidBodyByObject(left, {
        shape: "convex",
        collisionFlags: 2,
      });
      physicsManager.createRigidBodyByObject(right, {
        shape: "convex",
        collisionFlags: 2,
      });

      left.body.setFriction(1);
      right.body.setFriction(1);

      // Objects that need to be grabbed
      for (let i = 1; i <= 6; i++) {
        const obj = editor.scene.getObjectByName("电池" + i);
        physicsManager.createRigidBodyByObject(obj, {
          shape: "convex",
        });
      }
`

`

  //PhysicsManager.js

    import { AmmoPhysics } from "@enable3d/ammo-physics";
    import * as THREE from "three";
  
    export class PhysicsManager {
      constructor(editor) {
        this.editor = editor;
        this.physics = new AmmoPhysics(editor.scene);
        this.physics.debug.enable(true);
      }
    
      createRigidBodyByObject(object, options = {}) {
        this.physics.add.existing(object, options);
      }
    
      update(delta) {
        this.physics.update(delta * 1000);
        this.physics.updateDebugger();
      }
    }

`

**Comment by yandeu** [comment]
Have you looked at this example?  
https://enable3d.io/examples/standalone/pick-up-objects.html

I highly recommend using a box shape for the bricks.  
And a compound of box shapes for the crane.

Maybe you also want to change the **fixedTimeStep** in https://enable3d.io/docs.html#physics-configuration.

**Comment by volcanoqq** [comment]
Yes, I've seen the example of pick-up objects, but my animation used tween. 
If you use box shape to create, there will be an object rotation mismatch problem.
In this issue of mine at [(https://github.com/enable3d/enable3d/issues/365)](https://github.com/enable3d/enable3d/issues/365)
What does fixedTimeStep do, and why do I need to change it?

**Comment by yandeu** [comment]
> What does fixedTimeStep do

See the bullet manual:  
https://github.com/bulletphysics/bullet3/blob/master/docs/Bullet_User_Manual.pdf
